assets/sass/layout.scss

```diff
@@ -68,17 +68,48 @@ aside {
   border: solid 1px var(--main-border);
   @include border-radius(5px);
 
-  .two-column {
-    @include clearfix;
-
-    .column-left {
-      width: 47%;
-      float: left;
+  .video-grid {
+    padding: 1rem 0;
+
+    .video-thumbnails {
+      display: grid;
+      grid-template-columns: 1fr;
+      gap: 2rem;
+      list-style: none;
+      padding: 0;
+      margin: 0;
+
+      @media (min-width: 600px) {
+        grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+      }
     }
 
-    .column-right {
-      width: 47%;
-      float: right;
+    .video-card {
+      display: flex;
+      flex-direction: column;
+      text-decoration: none;
+      color: inherit;
+      height: 100%;
+      transition: transform 0.2s ease;
+
+      &:hover {
+        transform: translateY(-3px);
+      }
+
+      img {
+        width: 100%;
+        height: auto;
+        aspect-ratio: 16/9;
+        object-fit: cover;
+      }
+
+      h4 {
+        margin: 0.75rem 0;
+      }
```

Comment:
If we do here:

```css
&::before {
  content: counter(--video-index) ".";
  font-weight: normal;
}
```

And on `.video-thumbnails li` do `counter-increment: --video--index;`, we would have counters on the videos.

<img width="1368" height="746" alt="Image" src="https://github.com/user-attachments/assets/b0ca8819-50b2-4f3b-b77c-a2de0a2c6630" />